tests/test_client.py

```diff
@@ -746,3 +746,24 @@ def test_deposit_from_polygon_network_with_signer_low_balance():
     with pytest.raises(BalanceTooLowError):
         client.deposit_from_polygon_network_with_signer(signer=test_signer, provider=test_provider, amount=0.0001, currency='matic')
 
+@responses.activate
+def test_deposit_from_cross_chain_network_with_signer_invalid_amount():
+    w3 = Web3()
+    # test_signer = "0xb500b7238e2aFbBca678B2A3B462e35ab001A4a0"
+    test_signer = w3.eth.account.create()
+
+    test_provider = Web3(EthereumTesterProvider())
+
+    with pytest.raises(InvalidAmountError):
+        client.deposit_from_cross_network_with_signer(signer=test_signer, provider=test_provider, amount=0, currency='matic', network="POLYGON") 
+
+@responses.activate
+def test_deposit_from_cross_network_with_signer_low_balance():
```

Comment:
add test for success case

Reply:
the tests task is in queue for me, will add test for every functionality

Reply:
done